content/collections/experiment-apis/en/experiment-management-api-deployments.md

```diff
@@ -125,10 +125,10 @@ Edit a deployment that experiments or flags can be assigned to.
 
 ### Query parameters
 
-|Name|Description|
-|---|----|
-|`label`| Optional | Deployment's label. Must contain alphanumeric and/or `_`, `-` characters. |
-|`archive`| Optional | string | Soft delete or restore deployment. |
+|Name|Requirement|Type|Description|
+|---|----|----|----|
+|`label`| Optional | string | Deployment's label. Must contain alphanumeric and/or `_`, `-` characters. |
```

Comment:
<img width="886" alt="Screenshot 2024-12-17 at 5 31 50 PM" src="https://github.com/user-attachments/assets/cf00e981-031c-481f-a404-e01fb1a7f8e4" />

markdown was not written properly so it did not format the table correctly

Reply:
<img width="971" alt="Screenshot 2024-12-17 at 5 33 22 PM" src="https://github.com/user-attachments/assets/aad88c25-926e-4654-a19c-9c6c2ae52827" />